src/rdkafka_admin.c

```diff
@@ -2867,6 +2867,8 @@ const char *rd_kafka_ConfigSource_name(rd_kafka_ConfigSource_t confsource) {
             "UNKNOWN_CONFIG",        "DYNAMIC_TOPIC_CONFIG",
             "DYNAMIC_BROKER_CONFIG", "DYNAMIC_DEFAULT_BROKER_CONFIG",
             "STATIC_BROKER_CONFIG",  "DEFAULT_CONFIG",
+            "UNSUPPORTED",           "UNSUPPORTED",
```

Comment:
Let's put the real name, even if we don't have the configuration values still they could be returned by the broker.
```suggestion
            "DYNAMIC_BROKER_LOGGER_CONFIG", "CLIENT_METRICS_CONFIG",
```